### DISCUSSION TITLE
20240509 Weekly Development Meeting

### DISCUSSION THREAD

**Discussion by josecelano** [nautilus-cyberneering/.github#93] [Meetings]
- Time:  9:30 in Canary Islands (WET (UTC+0)).
- Time: 10:30 in Vaduz (CET (UTC+1)).

Location:
<https://meeting.blockfinance-eco.li/NautilusCyberneeringWeeklyDevMeeting>

Expected duration: 2 to 4 hours with breaks

Communicated attendants:

- Cameron Garnham(@da2ce7)  - 
- Jose Celano(@josecelano) - x
- Constantin Bosse(@cgbosse) - x
- Mario Melchor(@mario-nt) - x
- Graeme Byrne(@grmbyrn) - x
- Iván Santana(@ivan9989-pkj) - x
- Ismael Torrado(@Lepisma07) - x
- Borja (@BorjaPractica) - x

-------------------------------------------------------------

## Agenda:

1. Everyone will take their turn and:
- Go over his notes added as comments for the meeting which will have been added in advance to give an update of all the work done during the previous week indicating.
	- The issues worked on
	- The progress made on them
	- Other things done
- Inform of any faced challenges and request help if necessary to work on specific issues.
- Highlight the encountered difficulties, especially if there is a risk of them causing delays or affecting end product's quality.
- Communicate what will be the next week's tasks.
- List any new topics for further discussion in an additional Technical meeting that he noted for future discussion in comments to this discussion either before the meeting or after his turn so that they get considered.

During the meeting every team member will take their turns and follow this sequence.
The topics for further discussion will be noted in advance by each the team member and to the meetings agenda.

2. At the end of the meeting if there any new technical topics to be discussed further it will be decided to
set another meeting date for the discussion of these or to hold the meeting hereafter.

Also the persons to be part of the meeting will be decided on so that the others will be free to continue their own work.

**Comment by josecelano** [comment]
### Torrust Tracker

New issues:

- https://github.com/torrust/torrust-tracker/issues/831
- https://github.com/torrust/torrust-tracker/issues/842

Issues: 

- [x] https://github.com/torrust/torrust-tracker/issues/830
- [ ] https://github.com/torrust/torrust-tracker/issues/833
  - [x] https://github.com/torrust/torrust-tracker/issues/836 (update aquatic)
  - [x] https://github.com/torrust/torrust-tracker/issues/838 (update ringbuf)

PR:

- https://github.com/torrust/torrust-tracker/pull/808 (Migration to Figment) **WIP**
- https://github.com/torrust/torrust-tracker/pull/843

PR review:

- https://github.com/torrust/torrust-tracker/pull/814
- https://github.com/torrust/torrust-tracker/pull/826
- https://github.com/torrust/torrust-tracker/pull/827

### Torrust Index GUI

PR review:

- https://github.com/torrust/torrust-index-gui/pull/548

### Torrust Index GUI User Guide

New issues:

- [ ] https://github.com/torrust/torrust-index-gui-user-guide/issues/1

### Torrust Website

New issues:

- [ ] https://github.com/torrust/torrust-website/issues/175
- [ ] https://github.com/torrust/torrust-website/issues/176

PR:

- https://github.com/torrust/torrust-website/pull/173
- https://github.com/torrust/torrust-website/pull/174

PR review:

-  https://github.com/torrust/torrust-website/pull/172

**Comment by mario-nt** [comment]
### Index
	
- https://github.com/torrust/torrust-index/issues/448

- https://github.com/mario-nt/torrust-index/tree/casbin-test

### GUI
	
- https://github.com/torrust/torrust-index-gui/issues/447

### Training/Research

- Tidy first? book
- Role authorization research Casbin and Cerberos

**Comment by grmbyrn** [comment]
## Website
* Create folder for non-reusable components called `singletons` [#169](https://github.com/torrust/torrust-website/issues/169)
* Publish storybook on GitHub pages [#171](https://github.com/torrust/torrust-website/issues/171)
* Blog post tags should link to a list of post tagged with that tag [#175](https://github.com/torrust/torrust-website/issues/175)

## GPG
* Set up GPG key to sign commits

## Histoire
* https://www.youtube.com/watch?v=fWcuLqIhrrA&t=273s&ab_channel=AlvaroDevLabs

## New Technology
* SvelteKit revision
* Next.js
* Drizzle
* Clerk
* UploadThing
